Sources/OktaOidc/Common/OktaOidcError.swift

```diff
@@ -30,17 +31,49 @@ public enum OktaOidcError: Error {
     case noUserInfoEndpoint
     case parseFailure
     case missingIdToken
-    case unexpectedAuthCodeResponse
+    case unexpectedAuthCodeResponse(statusCode: Int)
     case userCancelledAuthorizationFlow
     case unableToGetAuthCode
     case redirectServerError(String)
+    /// See [RFC6749 Error Response](https://tools.ietf.org/html/rfc6749#section-4.1.2.1).
+    case authorization(error: String, description: String?)
+    case noLocationHeader
+    
+    public static var errorDomain: String = "\(Self.self)"
+    
+    public static let generalErrorCode = -1012009
```

Comment:
I don't know if we need to have some error code.
btw, 01/01/2009 is Okta' foundation date :D

Reply:
I'd make this internal or private.

Reply:
Agreed. Done